docs/integrations/microsoft-azure/web-apps.md

```diff
@@ -78,6 +88,130 @@ In Step 1, you create an HTTP source. When you configure the, plan your source c
+| parse regex field=resource_name "(?<parent_resource_name>[^/]+)(?:/PROVIDERS/[^/]+)?/(?<service_type>[^/]+)/?(?<service_name>.+)" nodrop
+| if (isBlank(parent_resource_name), resource_name, parent_resource_name) as resource_name
+| fields subscription_id, location, provider_name, resource_group, resource_type, resource_name, service_type, service_name
+```
+
+
+### Configure metric rules
+
+#### Azure Observability Metadata Extraction Service Level
+
+If this rule already exists, there's no need to create it again.
+
+```sql
+Rule Name: AzureObservabilityMetadataExtractionAppServiceLevel
+```
+
+```sql title="Metric match expression"
+resourceId=/SUBSCRIPTIONS/*/RESOURCEGROUPS/*/PROVIDERS/*/SITES/* tenant_name=*
+```
+
+| Fields extracted | Metric rule    |
+|:-----------------|:---------------|
+| `subscription_id`  | $resourceId._1 |
+| `resource_group`   | $resourceId._2 |
+| `provider_name`    | $resourceId._3 |
+| `resource_type`    | SITES |
+| `resource_name`    | $resourceId._4 |
+
+
+### Configure metrics collection
+
+In this section, you will configure a pipeline for shipping metrics from Azure Monitor to an Event Hub, on to an Azure Function, and finally to an HTTP Source on a hosted collector in Sumo Logic.
+
+1. [Configure an HTTP Source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-1-configure-an-http-source).
+1. [Configure and deploy the ARM Template](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-2-configure-azure-resources-using-arm-template).
+1. [Export metrics to Event Hub](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-3-export-metrics-for-a-particular-resource-to-event-hub). Perform below steps for each Azure WebApps that you want to monitor.
+   1. Choose `Stream to an event hub` as destination.
+   1. Select `AllMetrics`.
+   1. Use the Event Hub namespace created by the ARM template in Step 2 above. You can create a new Event Hub or use the one created by ARM template. You can use the default policy `RootManageSharedAccessKey` as the policy name.
+
+### Configure logs collection
+
+#### Diagnostic logs
+
+In this section, you will configure a pipeline for shipping diagnostic logs from [Azure Monitor](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-get-started) to an Event Hub.
+
+1. To set up the Azure Event Hubs source in Sumo Logic, refer to the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
+1. To create the **Diagnostic setting** in the Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-gb/azure/data-factory/monitor-configure-diagnostics). Perform the below steps for each Azure WebApps that you want to monitor.
+   1. Choose `Stream to an event hub` as the destination.
+   1. Select `AllMetrics`.
```

Comment:
this is logs section why AllMetrics?

Reply:
Updated with all the available log types.